### ISSUE TITLE
paused property ignored on Number Tween

### ISSUE THREAD

**Issue by marcusx2** [photonstorm/phaser#5730]
The [NumberTweenBuilderConfig](https://photonstorm.github.io/phaser3-docs/Phaser.Types.Tweens.html#.NumberTweenBuilderConfig) has a paused property that is completely ignored. When you use the [addCounter](https://photonstorm.github.io/phaser3-docs/Phaser.Tweens.TweenManager.html) method it creates a tween that autostarts by default, and setting the paused property to true has no effect. The workaround is to simply call the pause method immediately after addCounter.

**Comment by samme** [comment]
It worked for me in <https://labs.phaser.io/edit.html?src=src/tweens\number%20tween.js&v=3.55.2>.

**Comment by marcusx2** [comment]
Indeed. I don't know what's going on then. In my project the paused property doesn't work at all.

**Comment by photonstorm** [comment]
I tried this back to version 3.10 and it works ok, so am closing this off. Here's the test:

```js
    var tween = this.tweens.addCounter({
        from: 0,
        to: 100,
        duration: 5000,
        paused: true
    });

    this.input.once('pointerdown', () => {
        tween.resume();
    });
```
